yt/visualization/plot_modifications.py

```diff
@@ -769,6 +790,10 @@ def _finalize(self, plot, X, Y, pixX, pixY, pixC):
             "scale_units": self.scale_units,
         }
         kwargs.update(self.plot_args)
+
+        if plot._transform is not None:
+            kwargs["transform"] = plot._transform
```

Comment:
what if there's already a `transform` keyword argument ? As is, it would simply be silently dropped, which may be the source of subtle bugs down the road.

Reply:
added a warning.